frontend/src/utils/schedules.js

```diff
@@ -33,7 +33,10 @@ export const handleScheduleSave = async ({
       })
 
       params.api.forEachNode((rowNode) => {
-        if (rowNode.data.fuelSupplyId === newWarnings[0].id) {
+        if (
+          rowNode.data.fuelSupplyId === newWarnings[0].id ||
```

Comment:
Nice fix, I think this would work for all situations:
rowNode.data[idField] ==== newWarnings[0].id